crates/services/relayer/src/service/get_logs/test.rs

```diff
@@ -50,7 +50,7 @@ fn message(nonce: u64, block_number: u64, contract_address: u32, index: u64) ->
         ..Default::default()
     };
     let mut log = message.into_log();
-    log.address = u32_to_contract(contract_address);
+    log.address = H160::from_slice(u32_to_contract(contract_address).as_slice());
```

Comment:
Given that we're getting rid of references to `H160` it might be cleaner to use the `ethereum_types::Address` type alias here.

Reply:
Great idea, thank you ! I have made the modifications everywhere I has to use `H160`

Reply:
Nice, I think there's still one in `tests/tests/relayer.rs` though. Otherwise it looks good to me.
![image](https://github.com/user-attachments/assets/4ae97ce0-7cdb-447a-80e4-5fb0df56b9bc)

Reply:
I had bad filter on my IDE :/ Should be fine now, ty :)